apis/oam/v1alpha2/types.go

```diff
@@ -0,0 +1,857 @@
+/*
+Copyright 2019 The Crossplane Authors.
```

Comment:
We might need to revisit this part after the governance/trademark agreement is signed.

Reply:
also, it's 2020 now

Reply:
I'll defer to @bassam here. :)